core/src/main/java/org/stellar/anchor/config/Sep10Config.java

```diff
@@ -15,23 +15,20 @@ public interface Sep10Config {
   /**
    * The `web_auth_domain` property of <a
    * href="https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#response">SEP-10</a>.
-   * If the `web_auth_domain` is not specified, the `web_auth_domain` will be set to the domain of
-   * the value of the `home_domain`. `web_auth_domain` value must be equal to the host of the SEP
-   * server.
+   * If the `web_auth_domain` is not specified, the `web_auth_domain` will be set to the first value
+   * of `home_domains`. `web_auth_domain` value must be equal to the host of the SEP server.
```

Comment:
Will update this comment to align with the one in  `anchor-config-default-values.yaml `